CHANGELOG.md

```diff
@@ -56,6 +56,11 @@
 
   The configuration option `instrumentation.thread.tracing` is now enabled by default. This will allow the agent to properly monitor code that occurs inside of threads. If you are currently using custom instrumentation to start a new transaction inside of threads, this may be a breaking change, as it will no longer start a new transaction if one already exists.
 
+- **Add Fiber Instrumentation**
+
+  Fibers are now automatically instrumented similarly to threads. This can be configured using `instrumentation.fiber`.
```

Comment:
```suggestion
  `Fiber` instances are now automatically instrumented similarly to `Thread` instances. This can be configured using `instrumentation.fiber`.
```